www/install/step_upgrade/step3.php

```diff
@@ -48,7 +48,11 @@
 if (is_file('../RELEASENOTES.html')) {
     $contents = "<div id='releasenotes'></div>";
 } else {
-    $contents = "<textarea cols='100' rows='30' readonly>".file_get_contents('../RELEASENOTES')."</textarea>";
+    $releasenotesContent = '';
+    if (file_exists('../RELEASENOTES')) {
+        $releasenotesContent = file_get_contents('../RELEASENOTES');
+    }
+    $contents = "<textarea cols='100' rows='30' readonly>".$releasenotesContent."</textarea>";
```

Comment:
".$releasenotes-Content."<   ->     " . $releasenotesContent . "<

Reply:
Can you give me more details about your note?